vagrant-post-script/orchestrate_docker.sh

```diff
@@ -1,8 +1,4 @@
 #!/bin/bash
 
 cd oms-docker/docker
-docker-compose up -d
-echo "############# ok until here"
-docker-compose logs -f omscore-bootstrap
-docker-compose logs -f omsevents-bootstrap
-echo "############# done"
+make bootstrap
```

Comment:
Okay, I think you made your point and I understand the added value of vagrant :)